waspc/data/Generator/templates/react-app/src/entities/index.ts

```diff
@@ -9,6 +9,10 @@ export type {
   {=# entities =}
   {= name =},
   {=/ entities =}
+  {=# isAuthEnabled =}
```

Comment:
We inject the auth related entities by hand since their are not part of `AppSpec`